test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -124,6 +127,8 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			ginkgo.By("creating two workloads but delaying cluster queue creation which has enough capacity")
 			prodWl := testing.MakeWorkload("prod-wl", ns.Name).Queue(prodQueue.Name).Request(corev1.ResourceCPU, "11").Obj()
 			gomega.Expect(k8sClient.Create(ctx, prodWl)).Should(gomega.Succeed())
+			// wait a second to ensure the CreationTimestamps differ and scheduler picks the first created to be admitted
+			time.Sleep(time.Second)
```

Comment:
If I understand correctly, this is because when sorting the entries in scheduling cycle, they have the same timestamp by chance?

Reply:
Yes, and the resolution of the CreationTimestamp is 1s

Reply:
or you could validate how many workloads are admitted (via metrics) without committing to which one is admitted.

Reply:
I'm thinking that when working on the timeouts I may need to introduce a fake clock and then the test could use it.